src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,16 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        String[] mashine = new String[] {"Truck", "Bulldozer", "Excavator"};
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+        truck.doWork();
+        bulldozer.doWork();
+        excavator.doWork();
+        truck.stopWork();
+        bulldozer.stopWork();
+        excavator.stopWork();
```

Comment:
Make sure that the messages in the `doWork()` and `stopWork()` methods are informative and indicate what type of `Machine` is working or stopping work.